client/src/components/Atoms/CSVForm.tsx

```diff
@@ -127,15 +131,19 @@ const CSVFile = ({
               <>
                 <FormSection>
                   <FileInput
-                    inputProps={
-                      values.cvrFileType === CvrFileType.HART
-                        ? { accept: '.zip', name: 'zip' }
-                        : {
-                            accept: '.csv',
-                            name: 'csv',
-                            multiple: values.cvrFileType === CvrFileType.ESS,
-                          }
-                    }
+                    inputProps={{
+                      // While this component is named CSVFile, it can accept zip files in the case
+                      // of Hart CVRs
+                      // TODO: Consider renaming the component and its internals accordingly
```

Comment:
I forgot until I read this comment that I actually meant to deprecate this component. I built an improved file uploading component (that's currently only in use in the AA progress screen) called FileUpload that is intended to replace this on the JM screen. Made a task to track: https://github.com/votingworks/arlo/issues/1559

That makes me think we should probably also update the file uploader on the AA progress screen to also support this additional file type for Hart.

Reply:
Ooh great to know! The separation of concerns with the new FileUpload feels much cleaner

Gave audit admins the ability to upload scanned ballot information CSVs as well and added a corresponding screencapture to the PR description: https://github.com/votingworks/arlo/pull/1558/commits/55750e0d781155edbb86f48f62cb13a889087a04